server/server.go

```diff
@@ -108,7 +131,8 @@ func StartCredentialProxy() error {
 	return StartCredentialProxyWithSudo()
 }
 
-func StartCredentialsServer(creds *vault.VaultCredentials) error {
+func StartCredentialsServer(creds *vault.VaultCredentials, region string) error {
+	metadataRegion = region
```

Comment:
instead of using the global here it could easily be passed through the function calls as an argument